management/src/main/java/nearsoft/skytouch/management/controller/ProductController.java

```diff
@@ -23,7 +23,8 @@ public ProductController(ProductService productService) {
 
     @RequestMapping(value = "/view/product", method = RequestMethod.GET)
     private String newProductView(Map<String, Object> model) {
-        model.put("product", new Product());
+        Product emptyProduct = new Product();
```

Comment:
why did you prefer to create the variable?

Reply:
I changed this to denote that an empty product is passed to the view for fill it later